sound/soc/sof/ipc4.c

```diff
@@ -542,6 +542,20 @@ static int sof_ipc4_init_msg_memory(struct snd_sof_dev *sdev)
 	return 0;
 }
 
+static void sof_ipc4_create_exception_debugfs_node(struct snd_sof_dev *sdev)
+{
+	u32 telemetry_data_offset;
+
+	/* strip the first magic number dword */
+	telemetry_data_offset = sdev->debug_box.offset + sizeof(u32) +
+			(SOF_IPC4_DEBUG_TELEMETRY_SLOT_ID + 1) * SOF_IPC4_DEBUG_SLOT_SIZE;
```

Comment:
The slots are (by definition) dynamically allocated and the functionality can be in different slots on each DSP boot.
You cannot assume that telemetry is _always_ at this slot, especially if the telemetry is not even enabled in the firmware.

Reply:
@ujfalusi Only slot1 is used for telemetry when some DSP core gets FW panic in zephyr, so we don't need to  consider all of DSP core.
````c
static void coredump_mem_window_backend_start(void)
{
	/* Reset error & mem write ptr */
	error = 0;
	mem_wptr = 0;
	ADSP_DW->descs[1].type = ADSP_DW_SLOT_TELEMETRY;
````